### PR TITLE
Apparently there are machines reporting Windows 98 as well, this should be the last fix for these quirks

### PR DESCRIPTION
**Pull request by vdjeric**

### CODE REVIEW

File: telemetry/convert.py [hunk 1]
```diff
@@ -221,7 +221,9 @@ def convert_saved_session(self, json_dict):
 
         # WINNT is reported as Windows_NT in the unified ping, and apparently
         # we have some Win95 users
-        if info.get("OS") == "Windows_NT" or info.get("OS") == "Windows_95":
+        if info.get("OS") == "Windows_NT" or \
+           info.get("OS") == "Windows_95" or \
+           info.get("OS") == "Windows_98":
```
Comment (mreid-moz) [inline telemetry/convert.py]: Should we just use `info.get("OS").startswith("Windows_")`?

Comment (vdjeric) [inline telemetry/convert.py]: That works too, reject this pull request, I'll send a new one

Comment (mreid-moz) [inline telemetry/convert.py]: Just update this one :)

**Comment by vdjeric** [comment]
Updated

File: telemetry/convert.py [hunk 2]
```diff
@@ -221,7 +221,7 @@ def convert_saved_session(self, json_dict):
 
         # WINNT is reported as Windows_NT in the unified ping, and apparently
         # we have some Win95 users
-        if info.get("OS") == "Windows_NT" or info.get("OS") == "Windows_95":
+        if info.get("OS").startswith("Windows_"):
```
Comment (mreid-moz) [inline telemetry/convert.py]: Sorry, we should also guard for `None`, so let's use `info.get("OS", "Other").startswith("Windows_"):`

Comment (vdjeric) [inline telemetry/convert.py]: Good point

**Pull request merged**